src/components/deck-gl-wrapper/deck-gl-wrapper.tsx

```diff
@@ -344,15 +349,15 @@ export const DeckGlWrapper = ({
     });
   }, [loadTiles]);
 
-  useEffect(() => {
+  useMemo(() => {
```

Comment:
This change seems wrong. `useMemo` should return a value

Reply:
Done.